.travis/deploy.sh

```diff
@@ -16,8 +16,23 @@ if [[ "$TRAVIS_OS_NAME" == "linux" && ${BUILD_LINUX_WHEELS} -eq 1 ]]; then
     .travis/build_windows_wheels.sh
 else
 	if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-		python -m pip install wheel
-		python setup.py bdist_wheel
+		# Make sure we can build and "fix" the wheel.
```

Comment:
@ofek I'm not sure if you **want** a mix of tabs and spaces in this file, but I'm happy to convert all of the tabs as part of this PR.

Reply:
I didn't notice; yes please!

Reply:
Done.